src/main/java/com/google/cloud/flink/bigquery/FlinkBigQueryConfig.java

```diff
@@ -776,71 +789,11 @@ public int expirationTimeInMinutes() {
     };
   }
 
-  public enum IntermediateFormat {
-    AVRO("avro", FormatOptions.avro()),
-    AVRO_2_3("com.databricks.spark.avro", FormatOptions.avro()),
-    ORC("orc", FormatOptions.orc()),
-    PARQUET("parquet", FormatOptions.parquet());
-
-    private static Set<String> PERMITTED_DATA_SOURCES =
-        Stream.of(values())
-            .map(IntermediateFormat::getDataSource)
-            .filter(dataSource -> !dataSource.contains("."))
-            .collect(Collectors.toSet());
-
-    private final String dataSource;
-    private final FormatOptions formatOptions;
-
-    IntermediateFormat(String dataSource, FormatOptions formatOptions) {
-      this.dataSource = dataSource;
-      this.formatOptions = formatOptions;
-    }
-
-    public static IntermediateFormat from(
-        String format,
-        String flinkVersion,
-        org.apache.flink.configuration.Configuration sqlConf,
-        boolean validateSparkAvro) {
-      Preconditions.checkArgument(
-          PERMITTED_DATA_SOURCES.contains(format.toLowerCase()),
-          "Data write format '%s' is not supported. Supported formats are %s",
-          format,
-          PERMITTED_DATA_SOURCES);
-
-      if (validateSparkAvro && format.equalsIgnoreCase("avro")) {
-        IntermediateFormat intermediateFormat = isSpark24OrAbove(flinkVersion) ? AVRO : AVRO_2_3;
-        return intermediateFormat;
-      }
-
-      return Stream.of(values())
-          .filter(intermediateFormat -> intermediateFormat.getDataSource().equalsIgnoreCase(format))
-          .findFirst()
-          .get();
-    }
-
-    static boolean isSpark24OrAbove(String flinkVersion) {
-      return flinkVersion.compareTo("2.4") > 0;
-    }
-
-    public String getDataSource() {
-      return dataSource;
-    }
-
-    public FormatOptions getFormatOptions() {
-      return formatOptions;
-    }
-
-    public String getFileSuffix() {
-      return getFormatOptions().getType().toLowerCase();
-    }
-  }
-
-  private static class ParseSqlString {
+  class ParseSqlString {
```

Comment:
class names should be actors, not verbs. Please rename to `SqlParser`

Reply:
changed to SqlParser